src/EditorFeatures/Test/InheritanceMargin/InheritanceMarginTests.cs

```diff
@@ -989,6 +989,65 @@ public class {{|target1:Bar2|}} : IBar{lessThanToken}int{greaterThanToken}, IBar
                 itemForFooInBar2);
         }
 
+        [Fact]
+        public Task TestCSharpExplicitInterfaceImplementation()
+        {
+            var lessThanToken = SecurityElement.Escape("<");
+            var greaterThanToken = SecurityElement.Escape(">");
+            var markup = $@"
+interface {{|target2:IBar|}}{lessThanToken}T{greaterThanToken}
```

Comment:
i legit have no idea what's going on.  why is this not just `interface {{|target2:IBar|}}<T>`?

Reply:
I'm guessing this is eventually used in an XML definition for a project. Personally I'd be fine with just `interface {{|target2:IBar|}}&lt;T&gt;`

Reply:
yeah.  if this is eventually used in an xml definition, it shoudl be *that* code that escapes this code :)

Reply:
: ) Let me take a look and see if I can escape these char in the helper method